File: csharp/src/Slugify.MultiLang/SlugifySlugOptions.cs
using System.Text.RegularExpressions;

namespace Slugify.MultiLang;

public class SlugifySlugOptions
{
    public string Replacement { get; set; } = "-";
    public Regex? Remove { get; set; }
    public bool Lower { get; set; } = true;
    public bool Strict { get; set; } = true;
    public bool Trim { get; set; } = true;
    public string? Locale { get; set; }
}

File: java/src/main/java/io/github/balck3py/slugify/multilang/SlugifySlugOptions.java
package io.github.balck3py.slugify.multilang;

import java.util.regex.Pattern;

/** Options equivalent to the C# SlugifySlugOptions class. */
public class SlugifySlugOptions {
    private String replacement = "-";
    private Pattern remove;
    private boolean lower = true;
    private boolean strict = true;
    private boolean trim = true;
    private String locale;

    public String getReplacement() { return replacement; }
    public void setReplacement(String replacement) { this.replacement = replacement; }
    public Pattern getRemove() { return remove; }
    public void setRemove(Pattern remove) { this.remove = remove; }
    public boolean isLower() { return lower; }
    public void setLower(boolean lower) { this.lower = lower; }
    public boolean isStrict() { return strict; }
    public void setStrict(boolean strict) { this.strict = strict; }
    public boolean isTrim() { return trim; }
    public void setTrim(boolean trim) { this.trim = trim; }
    public String getLocale() { return locale; }
    public void setLocale(String locale) { this.locale = locale; }
}
